app/main.py

```diff
@@ -2,24 +2,75 @@
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(self, owner: type, name: str) -> None:
+        self.protected_name = f"_{name}"
+
+    def __get__(self, instance: object, owner: type) -> int:
+        return getattr(instance, self.protected_name)
+
+    def __set__(self, instance: object, value: int) -> None:
+        if not isinstance(value, int):
+            raise TypeError(f"{self.protected_name} should be an integer.")
+        if not (self.min_amount <= value <= self.max_amount):
+            raise ValueError(
+                f"{self.protected_name} should "
+                f"be in range {self.min_amount} "
+                f"to {self.max_amount}."
+            )
+        setattr(instance, self.protected_name, value)
 
 
 class Visitor:
-   pass
+    def __init__(self, name: str, age: int, weight: int, height: int) -> None:
+        self.name = name
+        self.age = age
+        self.weight = weight
+        self.height = height
```

Comment:
The `Visitor` class does not perform any validation on the input parameters. Consider using the `IntegerRange` descriptor for age, weight, and height fields to enforce data validation in the `Visitor` class as well.